### DISCUSSION TITLE
支持手动输入色值

### DISCUSSION THREAD

**Discussion by leatrise** [shion-app/shion#47] [Ideas]
如题，调色盘选色的时候难免会有偏差，（特别对于我这种有喜欢的色号的人来说）如果可以手动输入颜色的HEX色值或是设置RGB三原色的值会更方便
![image](https://github.com/user-attachments/assets/91c30672-5b57-4249-a203-db1ce12be09e)

**Comment by hanaTsuk1** [comment]
加入开发计划
